polartools/hkl_utils.py

```diff
@@ -21,22 +27,179 @@
     ~uan
     ~wh
     ~setlat
+    ~setaz
+    ~freeze
     ~update_lattice
-    ~read_config
     ~write_config
+    ~read_config
 """
 
+from hkl.util import Lattice
 import bluesky.plan_stubs as bps
 import pathlib
+import sys
+from instrument.framework import RE
+import fileinput
+import logging
 
 try:
     from hkl import cahkl
-    from hkl.user import _check_geom_selected, current_diffractometer
+    from hkl.user import (
+        _check_geom_selected,
+        _geom_,
+        select_diffractometer,
+        current_diffractometer,
+    )
     from hkl.configuration import DiffractometerConfiguration
+    from hkl.diffract import Diffractometer
+    from instrument.devices.polar_diffractometer import polar, polar_psi
```

Comment:
This doesn't work here. It will only work if we are in the correct environment in our Bluesky installation.